manifest.json

```
@@ -61,6 +61,7 @@
                   "public_302" : "Visible redirect (302, temporary). Everybody will be able to access it.",
                   "public_301" : "Visible redirect (301, permanent). Everybody will be able to access it.",
                   "public_proxy": "Proxy, invisible (nginx proxy_pass). Everybody will be able to access it.",
+				  "public_direct": "Proxy, invisible direct (nginx proxy_pass). Everybody will be able to access it. No forward headers.",
```

Just a small indentation problem here

```suggestion
                  "public_direct": "Proxy, invisible direct (nginx proxy_pass). Everybody will be able to access it. No forward headers.",
```